usr/src/test/test-runner/cmd/run.py

```diff
@@ -144,13 +145,16 @@ class Cmd(object):
     def __init__(self, pathname, outputdir=None, timeout=None, user=None):
         self.pathname = pathname
         self.outputdir = outputdir or 'BASEDIR'
```

Comment:
nit...
I'm not sure why the original author didn't set these defaults in the `def __init__` above, it is more friendly to do so and useful when using an IDE. Doing so avoids the need for the `or blah` here.